### PR TITLE
Howto investigate StackOverflowException in SLScripting

### PR DESCRIPTION
**Pull request by PedroDebevere**

### CODE REVIEW

File: user-guide/Troubleshooting/Procedures/SLScripting_StackOverflow.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+Open the dump file in Visual Studio, an press the *Run Diagnostic Analysis* link in the Actions list.
+A message box titled 'Exception Unhandled' appears with the following message:
+
+```txt
+Unhandled exception at 0x080BFA90 in CRASH_HIGH.DMP: 0xC00000FD: Stack overflow (parameters: 0x00000001, 0x0FA32FFC).
+```
+
+Now open the Parallel Stacks window (*Debug* > *Windows* > *Parallel Stacks*):
+
+![WinDbg](~/develop/images/VisualStudioParallelStacksWindow.png)<br>Visual Studio Parallel Stacks window*
```
Comment (SkylineThomasCR) [inline user-guide/Troubleshooting/Procedures/SLScripting_StackOverflow.md:20]:
```suggestion
![Visual Studio Parallel Stacks window](~/develop/images/VisualStudioParallelStacksWindow.png)<br>Visual Studio Parallel Stacks window*
```

File: user-guide/Troubleshooting/Procedures/SLScripting_StackOverflow.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+In this case, the parallel stacks window does not show a stack with huge number of calls making it overflow.
```
Comment (SkylineThomasCR) [inline user-guide/Troubleshooting/Procedures/SLScripting_StackOverflow.md:22]:
```suggestion
In this case, the parallel stacks window does not show a stack with huge number of calls that are typically of an overflow.
```

File: user-guide/Troubleshooting/Procedures/SLScripting_StackOverflow.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+Using WinDbg could possibly provide more insights.
+
+## WinDbg
+
+Open the dump file with WinDbg.
```
Comment (SkylineThomasCR) [inline user-guide/Troubleshooting/Procedures/SLScripting_StackOverflow.md:27]:
```suggestion
Open the dump file with [WinDbg](xref:WinDbg).
```

File: user-guide/Troubleshooting/Procedures/Troubleshooting_SLScripting_Issues.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+uid: TroubleshootingSLScriptingIssues
+---
+
+# Troubleshooting - SLScripting
```
Comment (SkylineThomasCR) [inline user-guide/Troubleshooting/Procedures/Troubleshooting_SLScripting_Issues.md:5]: #3587 does not link to a separate page, but puts the entire content in this page. Is this going to be aligned?

**Review by SkylineThomasCR — CHANGES_REQUESTED** [review]

File: user-guide/Troubleshooting/Procedures/SLScripting_StackOverflow.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
+---
+uid: TroubleshootingSLScriptingStackOverflowException
+---
+
+# Investigate StackOverflowException occurrence
+
+To illustrate how to investigate a [StackOverflowException](https://learn.microsoft.com/en-us/dotnet/api/system.stackoverflowexception?view=netframework-4.8.1) that occurs in SLScripting, an example connector has been created with a QAction that has a method that results in infinite recursion (i.e. a recursive method that does not have stop condition). DataMiner automatically created a dump and we will investigate the dump.
```
[the remark this answers is not in the document]
Comment (PedroDebevere) [reply user-guide/Troubleshooting/Procedures/SLScripting_StackOverflow.md:7]: Mention of directory has been added.

File: user-guide/Troubleshooting/Procedures/Troubleshooting_SLScripting_Issues.md (hunk 4 above)
Comment (PedroDebevere) [reply user-guide/Troubleshooting/Procedures/Troubleshooting_SLScripting_Issues.md:5]: yes, I'll adapt it  in the other PR

**Review by SkylineThomasCR — APPROVED** [review]

**Comment by MariekeGO** [comment]
@PedroDebevere, I've reviewed your PR and made some changes. Could you check if everything still looks OK to you?

**Comment by PedroDebevere** [comment]
@MariekeGO changes are OK

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**